### ISSUE TITLE
do not allow `mdmctl config set` without args

### ISSUE THREAD

**Issue by groob** [micromdm/micromdm#371]
currently it creates an empty entry

```
{
 "active": "",
 "servers": {
   "": {
     "api_token": "",
     "server_url": "",
     "skip_verify": false
   },
```

**Comment by jessepeterson** [comment]
I'd argue for allowing an empty name. Otherwise agree!

**Comment by scottknight** [comment]
@groob Should this get closed since #421 was merged?

**Comment by jessepeterson** [comment]
@scottknight FYI you can have PRs auto-close issues by including a "Fixed #<issuenum>": https://help.github.com/articles/closing-issues-using-keywords/

:)
